packages/lesswrong/components/posts/PostsPage/index.js

```diff
@@ -0,0 +1,8 @@
+import './PostsPageTwo.jsx'
+import './PostsAuthors.jsx'
+import './PostsPageTitle.jsx'
+import './PostsTopSequencesNav.jsx'
+import './PostsPageMetadata.jsx'
+import './HorizontalPostsVote.jsx'
+import './PostsPageMobileActions.jsx'
+import './PostActions.jsx'
```

Comment:
Is the idea to shift the convention here, to making `index.js` files for folders in the components tree rather than including them from `components.js`?